addons/website_slides_survey/static/src/js/public/components/slide_upload_dialog/slide_upload_dialog.js

```diff
@@ -0,0 +1,35 @@
+/** @odoo-module **/
+
+import { _t } from "@web/core/l10n/translation";
+import { SlideUploadDialog } from "@website_slides/js/public/components/slide_upload_dialog/slide_upload_dialog";
+import { patch } from "@web/core/utils/patch";
+
+patch(SlideUploadDialog.prototype, {
+    /**
+     * Overridden to add the "certification" slide category
+     */
```

Comment:
This and the other could instead be grouped in a single comment regarding the whole patch, or even be removed as the code is self-explanatory.